lib/litmus_paper/app.rb

```diff
@@ -33,6 +38,59 @@ class App < Sinatra::Base
       _create_status_file(StatusFile.global_health_file)
     end
 
+    get "/:service/status.json" do
+      health = _cache.get(params[:service])
+      _cache.set(
+        params[:service],
+        health = LitmusPaper.check_service(params[:service])
+      ) unless health
+
+      if health.nil?
+        _json 404, { "status" => "NOT FOUND" }, { "X-Health" => "0" }
+      else
+        json = {
+          :version => LitmusPaper::VERSION,
+          :name => params[:service],
+          :health => {},
+          :checks => {},
+          :dependencies => {}
+        }
+        if health.ok?
+          response_code = 200
+          status = "up"
+        else
+          response_code = 503
+          status = "down"
+        end
+        json[:status] = status
+        json[:response_code] = response_code
+
+        headers = {"X-Health" => health.value.to_s}
+        json[:health][:reported_health] = health.value
+        json[:health][:measured_health] = health.measured_health
+        if health.forced?
+          if health.direction == :health
+            status = "health"
```

Comment:
It looks like you set `status` here to `health`, but only use that in the `X-Health-Forced` status later on. Is that intentional, and/or congruent with the current behavior of the text endpoint?

Reply:
👍